docs/features/deserializing-events.md

```diff
@@ -4,14 +4,28 @@ layout: default
 ---
 
 ## Deserializing Events
-We provide support for deserializing events to typed objects:
+
+We provide support for deserializing events to typed event objects where the custom event payload is available via the `.GetPayload()` method.
+If you want to have the original raw JSON event payload, you can get it via the `.Data` property.
```

Comment:
Maybe a new line before this line would be good to break the paragraph?

![image](https://user-images.githubusercontent.com/4345663/65765506-b437ad80-e128-11e9-907f-bd450bc14150.png)

Just a suggestion!

Reply:
Yep.

Reply:
This is done, thanks!